spec/timezone.html

```diff
@@ -578,8 +567,12 @@ <h1>ToTemporalTimeZone ( _temporalTimeZoneLike_ )</h1>
           1. Set _temporalTimeZoneLike_ to ? Get(_temporalTimeZoneLike_, *"timeZone"*).
           1. If Type(_temporalTimeZoneLike_) is Object and ? HasProperty(_temporalTimeZoneLike_, *"timeZone"*) is *false*, return _temporalTimeZoneLike_.
         1. Let _identifier_ be ? ToString(_temporalTimeZoneLike_).
-        1. Let _result_ be ? ParseTemporalTimeZone(_identifier_).
-        1. Return ? CreateTemporalTimeZone(_result_).
+        1. Let _parseResult_ be ? ParseTemporalTimeZoneString(_string_).
+        1. If _parseResult_.[[Name]] is not *undefined*, then
+          1. If ! IsValidTimeZoneName(_parseResult_.[[Name]]) is *false*, throw a *RangeError* exception.
+          1. Return ! CreateTemporalTimeZone(! CanonicalizeTimeZoneName(_parseResult_.[[Name]])).
+        1. If _result_.[[Z]] is *true*, return ! CreateTemporalTimeZone(*"UTC"*).
```

Comment:
Should this be 
```
1. If _parseResult_.[[Z]] is *true*, return ! CreateTemporalTimeZone(*"UTC"*).
```
since _result_ is renamed to _parseResult_ in your PR now.